.github/pre-commit

```diff
@@ -8,6 +8,10 @@
 
```

Comment:
> Some thoughts:

- I am not sure about the placement of the file inside hack. Didn't we say, we want a dedicated doc(s) package one level above? I may be wrong there. I think @andriisoldatenko knows more there.
- I think you forgot to add the feature flags to the documentation, as far as i can see there are only e2e tests docu
- Would it be a good idea to maybe add a step where the tool gomarkdoc is downloaded and installed? For the case that one wants to run the make targets but does not have the tool installed?

I am quoting this here such that you guys can also comment :sweat_smile:

Reply:
- Moved scripts to ./doc/script as mentioned by you guys :+1: 
- Added script for feature_flags.go documentation efedfe13542ff8b80dd9dc815609578100e206c4
- Make command for installing gomarkdoc is already in place! `make prerequisites/gomarkdoc`